anchore_engine/db/entities/upgrade.py

```diff
@@ -1955,6 +1960,52 @@ def db_upgrade_013_014():
     upgrade_oauth_client_014()
 
 
+def remove_incorrect_github_vuln_data():
+    """
+    Github feed data was being processed under incorrect types 'github:os' and 'github:unknown'. This function removes
+    those vulnerability records synced by the policy engine because they are incorrect data
+    """
+    engine = anchore_engine.db.entities.common.get_engine()
+
+    namespace_names = ["github:os", "github:unknown"]
+
+    to_delete = (
+        (
+            ImagePackageVulnerability,
+            ImagePackageVulnerability.vulnerability_namespace_name,
+        ),
+        (VulnerableArtifact, VulnerableArtifact.namespace_name),
+        (FixedArtifact, FixedArtifact.namespace_name),
+        (Vulnerability, Vulnerability.namespace_name),
+        (FeedGroupMetadata, FeedGroupMetadata.name),
+    )
+
+    for type, column in to_delete:
+        log.err(
+            "Removing records from table {} where {} is {}".format(
+                type, column, namespace_names
+            ),
+        )
+
+        try:
+            with session_scope() as session:
+                results = session.query(type).filter(column.in_(namespace_names)).all()
```

Comment:
(nit) it would be nice to have log statement for how many records are getting deleted. It has to be at an error level to show up in the logs since it's executed prior to bootstrap. Something like 
```
log.err(f"Deleting {len(results)} records from {type.__tablename__}")
```